cpp/lib/src/ZoteroHarvesterConfig.cc

```diff
@@ -542,11 +542,13 @@ bool ParseExpectedLanguages(const std::string &expected_languages_string, Langua
     std::string expected_languages(expected_languages_string);
     if (expected_languages[0] == '*')
         return false;
-    else if (expected_languages[0] == '!' or expected_languages[0] == '?') {
+    else if (expected_languages[0] == '!' or expected_languages[0] == '?' or expected_languages[0] == '=') {
```

Comment:
If you add a new mode here, please don't forget to add it to the documentation section in zotero_harvester.conf after the PR is merged.